### DISCUSSION TITLE
How to add metric to gateway using post man

### DISCUSSION THREAD

**Discussion by lakshmisivareddy** [prometheus/pushgateway#465] [General]
Hi ,
i used below command  to run the gateway 
docker pull prom/pushgateway
docker run -d -p 9091:9091 prom/pushgateway

i am trying to add metrics in below formate 
 |Metrics Name  { Metrics Lables }  Metrics Vaule | 

Example :
some_metric{job="some_job"} 3.14

i tried below curl command and i am able to add a metric
  echo "some_metric 3.14" | curl --data-binary @- http://pushgateway.example.org:9091/metrics/job/some_job

but  i was not able to add same metrics using post man 

can some one help me with example how to add this metric using post man

**Comment by beorn7** [comment]
Could you explain what you mean by "using post man"?

**Comment by karthikbhat0307** [comment]
```
curl --location 'http://<push-gateway-url>/metrics/job/some_job/instance/some_instance' \
--header 'Content-Type: text/plain' \
--data-binary '/tmp/test-data.txt'
```
Above curl you can import in postman.
If you want to directly choose the file then go to body section and then choose binary and select your file path.

The below content can be your content of the text file /metric
```
# TYPE some_metric counter
some_metric{label="val1"} 42
# TYPE another_metric gauge
# HELP another_metric Just an example.
another_metric 2398.283
```
